inference.Rmd

```diff
@@ -431,44 +544,61 @@ grid.arrange(annotated_sampling_dist_20,
 )
 ```
 
-Based on the visualization in Figure \@ref(fig:11-example-means7), two points about the sample mean become clear.
-First, the
-mean of the sample mean (across samples) is equal to the population mean. In other words, 
-the sampling distribution is centred at the population mean.
-Second, increasing the size of the sample
-decreases the spread (i.e., the variability) of the sampling distribution. Therefore, a larger sample size results
-in a more reliable point estimate of the population parameter.
+Based on the visualization in Figure \@ref(fig:11-example-means7), two points
+about the sample mean become clear.  First, the mean of the sample mean (across
+samples) is equal to the population mean. In other words, the sampling
+distribution is centred at the population mean.  Second, increasing the size of
+the sample decreases the spread (i.e., the variability) of the sampling
+distribution. Therefore, a larger sample size results in a more reliable point
+estimate of the population parameter.
 
 
 <!--- > **Note:** If random samples of size $n$ are taken from a population, the sample mean $\bar{x}$ will be approximately Normal with mean $\mu$ and standard deviation $\frac{\sigma}{\sqrt{n}}$ as long as the sample size $n$ is large enough. $\mu$ is the population mean, $\sigma$ is the population standard deviation, $\bar{x}$ is the sample mean, and $n$ is the sample size. 
 > If samples are selected from a finite population as we are doing in this chapter, we should apply a finite population correction. We multiply $\frac{\sigma}{\sqrt{n}}$ by $\sqrt{\frac{N - n}{N - 1}}$ where $N$ is the population size and $n$ is the sample size. If our sample size, $n$, is small relative to the population size, this finite correction factor is less important. 
 ---> 
 
 ### Summary
-1. A *point estimate* is a single value computed using a sample from a population (e.g. a mean or proportion)
-2. The *sampling distribution* of an estimate is the distribution of the estimate for all possible samples of a fixed size from the same population.
-3. The sample means and proportions calculated from samples are centred around the population mean and proportion, respectively.
+1. A point estimate is a single value computed using a sample from a population (e.g. a mean or proportion)
+2. The sampling distribution of an estimate is the distribution of the estimate for all possible samples of a fixed size from the same population.
+3. The shape of the sampling distribution is usually bell-shaped with one peak and centred at the population mean or proportion.
```

Comment:
Perhaps we should add something to this sentence? Some kind of hedging (e.g. “if our random sample is large enough” type of thing)

Reply:
added a note box above it referencing figrue 10.7!